### ISSUE TITLE
Browser hangs forever while trying to process pattern

### ISSUE THREAD

**Issue by Kirkman** [Pictogrammers/bitmask-to-svg#1]
This library looks super-promising for my use case ... I'm trying to turn 16x16 binary fill patterns into a fontset. 

I can run the demo code from the repo (the 5x7 letter "P"), and it works. 

But every time I try to feed one of my patterns to bitmaskToPath, the browser hangs forever and never spits out any SVG code. I can't figure out what's wrong. 

The following example code will demonstrate the issue:

```
const pattern = [
	[1,0,0,0,1,0,0,0,1,0,0,0,1,0,0,0],
	[0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1],
	[0,0,1,0,0,0,1,0,0,0,1,0,0,0,1,0],
	[0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1],
	[1,0,0,0,1,0,0,0,1,0,0,0,1,0,0,0],
	[0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1],
	[0,0,1,0,0,0,1,0,0,0,1,0,0,0,1,0],
	[0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1],
	[1,0,0,0,1,0,0,0,1,0,0,0,1,0,0,0],
	[0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1],
	[0,0,1,0,0,0,1,0,0,0,1,0,0,0,1,0],
	[0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1],
	[1,0,0,0,1,0,0,0,1,0,0,0,1,0,0,0],
	[0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1],
	[0,0,1,0,0,0,1,0,0,0,1,0,0,0,1,0],
	[0,1,0,1,0,1,0,1,0,1,0,1,0,1,0,1]
];

const width = pattern[0].length;
const height = pattern.length;
console.log(pattern);
const path = bitmaskToPath(pattern);
console.log(width, height, path);
```

**Comment by Templarian** [comment]
@Kirkman I fixed this locally and thought I pushed the fix out to this repo a long time ago.

```js
 else if (data[0] instanceof Array) {
        bitmask = data.flat(); // <--- flattened already
        width = data[0].length; // <--- correct 16
        height = bitmask.length; // <--------- 256 height 🤦, instead of data.length which it's supposed to be.
    }
```

Wrote the library for the exact same use case I'm sure 99% of others will. Pixel font generation. We're working on an open source pixel / font editor at the moment.

One moment just got home. Grabbing some food then will push a fix out.

**Comment by Templarian** [comment]
Fixed in the demo and the repo.

Note: This was never an issue in the `@pictogrammers/bitmask-to-svg` NPM package as it already had the fix in `0.9.1`. 

Thanks @Kirkman for letting me know.
